src/_data/sidenav.yml

```
@@ -1,16 +1,45 @@
-- title: Get started
+- header: Get started
+- title: Set up Flutter
+  permalink: /get-started/install
+
+- title: Learn Flutter
   children:
-    - title: Install Flutter
-      permalink: /get-started/install
-    - title: Test drive
-      permalink: /get-started/test-drive
+    - title: Introduction
+      permalink: /get-started/learn-flutter
     - title: Write your first app
       permalink: /get-started/codelab
-    - title: Learn more
-      permalink: /get-started/learn-more
-    - divider
-    - title: Where to get Flutter support
-      permalink: /resources/support
+    - title: Fundamentals course
```

Should we adjust the title here too?

```suggestion
    - title: Learn the fundamentals
```